src/releng_docs/deploying.rst

```diff
@@ -1,146 +1,244 @@
 Deploying
 =========
 
-Merging to either ``staging`` or ``production`` branches will, if build is
-successfull, also trigger a deployment script. Deployment script will only
-deploy services that were changed, since last deployment.
+Deployment happens **automatically** in :ref:`continuous integration
+<continuous-integration>` when code is pushed / merged to **staging** or
+**production** branch.
 
-``mozilla-releng/services`` build system is flexible enough to deploy to many
-different targets. Currently we deploy services to Heroku and Amazon S3, but
-this can be extended to any other target.
+If interested please read more on :ref:`branching policy <branching-policy>`.
 
-Deploying of new services - at this time - does not handle creation/removal of
-S3 buckets / Heroku applications. Those need to be created beforehand.
 
-Manually deploying
-------------------
+Release Schedule
+----------------
 
-To deploy a single service manually, you need to run:
+Releases happen on a **weekly** basis, excluding `Firefox release week`_. `Next
+release`_ is announced in github issue tracker.
 
-.. code-block:: bash
 
-    % make deploy-staging APP=releng_clobberer ...
-        or
-    % make deploy-production APP=releng_clobberer ...
+Any **out-of-schedule** release can be requested by contacting :ref:`release
+manager <release-managers>` via irc or `creating a ticket on github`_.
 
-Above commands are missing many secrets which you need to provide as variables
-to make commands above. You can check how this is done on taskcluster via
-``.taskcluster.sh`` script.
+.. _`creating a ticket on github`: https://github.com/mozilla-releng/services/issues/new
+.. _`Next release`: https://github.com/mozilla-releng/services/issues?q=is%3Aopen+is%3Apr+label%3A%220.kind%3A+release%22
+.. _`Firefox release week`: https://wiki.mozilla.org/RapidRelease/Calendar
 
-Secrets are retrived from `taskcluster secrets services`_ for each branch
-separatly.
 
-- staging => `repo:github.com/mozilla-releng/services:branch:staging`_
-- production => `repo:github.com/mozilla-releng/services:branch:production`_
+Release Protocol
+----------------
 
+This protocal is followed by the :ref:`release manager <release-managers>` of ``mozilla-releng/services``.
```

Comment:
I think it might be best to avoid the word "release manager" here, it could easily make people think you're talking about the Release Management team.